config/ldp/metadata-parser/default.json

```diff
@@ -4,7 +4,8 @@
     "files-scs:config/ldp/metadata-parser/parsers/content-type.json",
     "files-scs:config/ldp/metadata-parser/parsers/content-length.json",
     "files-scs:config/ldp/metadata-parser/parsers/slug.json",
-    "files-scs:config/ldp/metadata-parser/parsers/link.json"
+    "files-scs:config/ldp/metadata-parser/parsers/link.json",
```

Comment:
Could you make this list alphabetical? Same for the list below. I realize it was already wrong before but this is a good opportunity to clean this up.